manifest.config.ts

```diff
@@ -86,17 +83,21 @@ export const firefoxManifest: Manifest.WebExtensionManifest = {
 };
 
 /**
- * Gets icon paths for a browser
+ * Gets action with defaults for a browser
  *
- * @param browser - browser to get icons for
- * @returns object of icon paths
+ * @param browser - browser to get action for
+ * @returns manifest action object
  */
-function browserIcons(browser: string) {
+function getAction(browser?: string) {
+	const type = browser === 'safari' ? 'safari' : 'light';
```

Comment:
This reads strange - so the type is either `safari` or `light`?

Reply:
Yes, I could probably name this better, basically this gets the type of the default icon. We cant know what theme the user has here so on the non safari browsers we simply have to pick either light or dark. It will change to the users selected theme once any navigation event happens that causes web scrobbler to refresh icon

Reply:
Updated this to be clearer